src/widgets/about-school/ui/about.scss

```diff
@@ -19,27 +19,6 @@
         flex-direction: column;
         gap: 16px;
         min-width: 300px;
-
-        > .paragraph {
-          @include media-laptop {
-            font-size: 18px;
-            line-height: 28px;
-          }
-
-          @include media-tablet {
-            font-size: 16px;
-            line-height: 24px;
-          }
-        }
-
-        @include media-laptop {
```

Comment:
Haven't we add this in `paragraph.module.scss`?

Reply:
[Resolved](https://github.com/rolling-scopes/site/pull/475/commits/c0158c42b2443716e3e9157717628ee7e2503cb4).